content/id/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Disrupsi
+content_template: templates/concept
+weight: 60
+---
+
+{{% capture overview %}}
+Petunjuk ini diperuntukkan bagi pemilik aplikasi yang ingin membuat aplikasi yang memiliki ketersediaan yang tinggi, sehingga dibutuhkan untuk mengerti jenis-jenis Gangguan yang dapat terjadi pada Pods.
+
+Petunjuk ini juga diperuntukkan bagi administrator kluster yang ingin melakukan berbagai tindakan otomasi pada kluster, seperti pembaruan dan *autoscaling* kluster.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Disrupsi yang Disengaja dan Tidak Disengaja
+
+Pods tidak akan terhapus hingga sesuatu (orang maupun *controller*) menghancurkan mereka atau ada kesalahan perangkat keras maupun perangkat lunak yang tidak dapat dihindari.
+
+Kita menyebut kasus-kasus yang tidak dapat dihindari sebagai *disrupsi yang tidak disengaja* terhadap aplikasi. Beberapa contohnya adalah sebagai berikut:
+
+- Kesalahahan perangkat keras pada mesin yang menjalankan Node
+- Administrator kluster menghapus *virtual machine* secara tidak sengaja
+- Kesalahan pada penyedia layanan cloud yang mengakibatkan terhapusnya *virtual machine*
+- Sebuah *kernel panic*
+- Node menghilang dari kluster karena partisi jaringan kluster
+- Diusirnya Pod karena Node [kehabisan sumber daya](/docs/tasks/administer-cluster/out-of-resource)
+
+Dengan pengecualian pada kondisi kehabisan sumber daya, kondisi-kondisi tersebut pada umumnya diketahui oleh kebanyakan pengguna; Kondisi-kondisi tersebut tidak hanya terjadi pada Kubernetes
+
+Kita menyebut kasus-kasus lainnya sebagai *disrupsi yang disengaja*. Hal ini termasuk tindakan yang dilakukan oleh pemilik aplikasi atau yang dilakukan oleh administrator kluster. Pemilik aplikasi umumnya melakukan hal-hal berikut:
+
+- Menghapus Deployment atau pengontrol yang mengatur Pod
```

Comment:
nit: pengontrol -> kontroler?

Reply:
It's not in the kbbi though https://kbbi.kemdikbud.go.id/entri/kontroler
Should we use _controller_ instead?
.
.
.
Or, speculate that kontroler will be added to the next kbbi revisions and use this now for practical reason? :smile: 
https://kbbi.kemdikbud.go.id/Beranda/Pemutakhiran

Reply:
_Pengendali_ is also a good candidate

Reply:
actually **pengendali** is in the RSNI (glosarium), so it's a good translation of controller which will be in KBBI soon. but for now let's use *controller* since it's there in most translated pages and aligned with Kubernetes terms